### DISCUSSION TITLE
RSS Feed link 404 is jumpped from Docu page

### DISCUSSION THREAD

**Discussion by tisonkun** [facebook/docusaurus#8718] [Q&A] [answered]
```
to: '/issue/atom.xml'
```

Gives 404 Not Found. cc @slorber 

You can see https://pulsarweekly.vercel.app/ click Feed links.

**Comment by tisonkun** [comment, marked as answer]
Fixed by forcing `target=_blank` for a tag.

**Reply by slorber** [reply]
The docusaurus link assumes by default you want to navigate to an internal / single-page-application route, unless there's a full domain name or a target !== self and some other rules. 

`/issues/atom.xml` is a valid frontend route URL so we can't really be sure how to handle this link.

Another solution would be to use a `<a>` tag instead of `<Link>`

**Reply by tisonkun** [reply]
https://github.com/tisonkun/pulsarweekly/commit/f7c52ae5a6efe70974403ed4d0c781af5789cad8

Here is my solution - just sharing.

(Out of the thread, what's the whitespace rule for JSX with line breaks in a single tag? I found that sometimes my content stuck together and sometimes there is whitespace properly.)
